React_assignments/react_2nd_module/todolist_practise/src/components/TodoApp.jsx

```diff
@@ -0,0 +1,74 @@
+import { AddTodoForm } from "./AddTodoForm";
+import { TodoFilter } from "./TodoFilter";
+import { TodoList } from "./TodoList";
+import { useCallback, useEffect, useState } from "react";
+import { useFetch } from "./useFetch";
+
+export const TodoApp = () => {
+  const [showCompleted, setShowCompleted] = useState(false);
+  const [addTodoInput, setAddTodoInput] = useState("");
+
+  const getTodos = useCallback(async () => {
+    const response = await fetch("https://jsonplaceholder.typicode.com/todos");
```

Comment:
Use can use .then method also to retrieve the response.json() as response.json() also return promise